tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/workspace/BundleUrlValidationService.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.gradle.core.workspace;
+
+import org.eclipse.sapphire.modeling.Status;
+import org.eclipse.sapphire.services.ValidationService;
+
+/**
+ * @author Terry Jia
+ */
+public class BundleUrlValidationService extends ValidationService
+{
+
+    @Override
+    protected Status compute()
+    {
+        Status retval = Status.createOkStatus();
+
+        String bundleUrl = op().getBundleUrl().content();
+
+        if( !bundleUrl.startsWith( "http" ) )
```

Comment:
We should only check to make sure that this is a valid URL.  Lets not worry about if it is "file" or "http" or whatever.